src/App.tsx

```diff
@@ -67,6 +74,12 @@ const App: React.FC = () => {
       <GlobalCheckClaimStatus excludeLocations={['/collectibles']} />
       <Menu>
         <SuspenseWithChunkError fallback={<PageLoader />}>
+          <Route path="/info">
+            <ProtocolUpdater />
```

Comment:
there is a update component on the root file

Reply:
yeah but if I put it in there then info data will be requested on all pages. Here info updaters get unmounted when not on info page.

Reply:
Alternatively we can put it in the root file inside `views/Info`

Reply:
There's no root file though, its separate pages. But I agree that it might be better to refactor it to have one root file, that would also make lazy loading better (now each section loads separately, but most of the time users probably will click through all of them). Will refactor that 👍